src/pubvis.js

```diff
@@ -535,6 +539,37 @@ PUBVIS = function () {
                 //console.log( "keywords.length: " + keywords.length );
             }
 
+            exclude_chosen_keywords = function (keywords) {
+                if(exclude_chosen_keywords.length > 0) {
+                    var tmp_keywords = [];
+    
+                    for(var i = 0; i < keywords.length; i++) {
+                        if(!lockForPush) {tmp_keywords.push(keywords[i-1])}
+                        var lockForPush = false;
+    
+                        exclude_keywords.forEach((word, index) => {
+                            console.log('compare ', word, '  to ', keywords[i].text);
+                            var tmp_keyword = keywords[i].text.toLowerCase();
+                            var tmp_regex = new RegExp(word);
+                            if(tmp_regex.test(tmp_keyword)) {
+                                console.log('found match at: ', tmp_keyword);
+                                lockForPush = true;
+                            }
+                        });
+    
```

Comment:
move the `push` here (don't forget to update array index):
 if(!lockForPush) {tmp_keywords.push(keywords[i])}